content/docs/command-reference/import-url.md

```diff
@@ -125,12 +125,15 @@ source.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
-- `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
-  the file or directory already exists, this can be used to skip the download.
-  The data hash is not calculated when this option is used, only the import
-  metadata is saved to the `.dvc` file. `dvc commit <out>.dvc` can be used if
-  the data hashes are needed in the `.dvc` file, and to save existing data to
-  the cache.
+- `--no-exec` - create the import `.dvc` file without actually downloading
+  `url`. No connection to the source location is made, so the data hash is not
```

Comment:
`No connection to the source location is made, so the data hash is not calculated.`

I'm not sure I get this tbh

Reply:
I simplified this in 2785d594. PTAL